Feat/shape 940 (#60)
storyblok/app-extension-auth: README.md

Reviewer: Thanks for documenting it @ademarCardoso, you are an angel 🫂

const href = `/my/other/page?spaceId=${spaceId}&userId=${userId}`
```

> ## How to run this application to run in localhost

Reviewer 2: ```suggestion
## How to run this application locally
```

const href = `/my/other/page?spaceId=${spaceId}&userId=${userId}`
```

## How to run this application locally

> To run OAuth locally, add `customAuthEndpoint` to the paramns object in the target project (i.e. the plugin or application you want to test). With this parameter, you can select the target's backend.

Reviewer 2: ```suggestion
To run OAuth locally, pass the `customAuthEndpoint` property to the params object when calling the `authHandler` function in the target project (i.e. the plugin or application you want to test). With this parameter, you can change the target's backend environment.
```

```typescript
import { AuthHandlerParams } from '@storyblok/app-extension-auth'

> export const params: AuthHandlerParams = {
>   clientId: process.env.APP_CLIENT_ID,
>   clientSecret: process.env.APP_CLIENT_SECRET,
>   baseUrl: process.env.APP_URL,
>   successCallback: '/',
>   errorCallback: '/401',
>   endpointPrefix: '/api/connect',
>   customAuthEndpoint: 'http://localhost:1234'
> }

Reviewer 2: ```suggestion
import { authHandler } from '@storyblok/app-extension-auth'

const params: AuthHandlerParams = {
  // ...
  customAuthEndpoint: 'http://localhost:1234'
}

authHandler(params)
```
